format.sh

```diff
@@ -54,6 +54,14 @@ YAPF_EXCLUDES=(
     '--exclude' 'sky/skylet/providers/ibm/**'
 )
 
+ISORT_YAPF_EXCLUDES=(
+    '--sg' 'build/**'
+    '--sg' 'sky/skylet/providers/aws/**'
+    '--sg' 'sky/skylet/providers/gcp/**'
+    '--sg' 'sky/skylet/providers/azure/**'
```

Comment:
(See other comment) Some files under these dirs seem formatted.

Reply:
I added isort with the `black` profile (L120) since they were already formatted by black (L105-106).
They are formatted but it seems that mostly things that were moved were `sky` imports.

Reply:
yes will do